packages/lab/src/form-field-next/FormFieldLabel.tsx

```diff
@@ -0,0 +1,33 @@
+import { clsx } from "clsx";
+import { HTMLAttributes } from "react";
+import { Label, makePrefixer } from "@salt-ds/core";
+
+import "./FormFieldLabel.css";
+
+const withBaseName = makePrefixer("saltFormFieldLabel");
+export interface FormFieldLabelProps extends HTMLAttributes<HTMLLabelElement> {
+  /**
+   * Whether the Form Field is disabled.
+   */
+  disabled?: boolean;
+  /**
+   * The label value for the FormFieldLabel
+   */
+  label?: string;
+}
+
+export const FormFieldLabel = ({
+  className,
+  disabled,
+  label,
+  ...restProps
+}: FormFieldLabelProps) => (
+  <Label
```

Comment:
why not use variant and disabled props?

Reply:
variant is always secondary - have improved and used disabled prop